makefile

```diff
@@ -109,6 +109,17 @@ pylint_py_files := \
     $(wildcard testsuite/test*.py) \
     testsuite/validate.py \
 
+# flake8 config file
+flake8_rc_file := flake8.rc
+
+# PyLint source files to check
+flake8_py_files := \
+    setup.py \
+    os_setup.py \
+    $(filter-out $(moftab_files), $(wildcard $(package_name)/*.py)) \
+    $(wildcard testsuite/test*.py) \
```

Comment:
We have files in `testsuite` that do not start with `test*` and that should be checked. Change `test*.py` to `*.py`.